components/feature/share/src/main/java/mozilla/components/feature/share/RecentAppsStorage.kt

```diff
@@ -37,6 +37,17 @@ class RecentAppsStorage(context: Context) {
         }
     }
 
+    /**
+     * Deletes an app form the recent apps list
+     * @param activityName - name of the activity of the app
+     */
+    fun deleteRecentApp(activityName: String) {
```

Comment:
It will be nice to have a test for this, you could find some similar tests [here](https://github.com/mozilla-mobile/android-components/blob/master/components/feature/share/src/test/java/mozilla/components/feature/share/RecentAppStorageTest.kt) :)

Reply:
Added the test for RecentAppsStorage.deleteRecentApp